UI Improvements

nmanumr opened the pull request:
Added few UI changes to header toolbar.

For Desktop:

![image](https://user-images.githubusercontent.com/19629102/47406307-07fa8380-d76f-11e8-99cc-226710d1760f.png)

For Mobile:

![image](https://user-images.githubusercontent.com/19629102/47406331-28c2d900-d76f-11e8-9399-70e3a12b0418.png)

Mobile Search active:

![image](https://user-images.githubusercontent.com/19629102/47406346-47c16b00-d76f-11e8-9408-be900e49bf84.png)

allanday commented:
I see vertical alignment issues in those screenshots - elements aren't centered. I'd also suggest sticking with all caps for the menu items.

Issue #54 has some relevant design work.

nmanumr commented:
@allanday Are u talking about this loading spinner or the flathub icon??

splitt3r commented:
It looks a lot better :). Awesome 😍

allanday commented:
> @allanday Are u talking about this loading spinner or the flathub icon??

The Flathub icon is the most noticable issue - the vertical alignment is 3/4px off. The search box and its content also looks slightly off-center, to my eyes.

nmanumr commented:
The feels a little off but it is not actually off. See here:

![image](https://user-images.githubusercontent.com/19629102/47585849-f46e3900-d977-11e8-87fe-ba849229b5b6.png)

We have the spacing of 14px from top and 15px from bottom.

and search box have 1px shift with spacing of 13px top and 16px bottom.

nmanumr commented:
I also changed toolbar links to uppercase

nmanumr commented:
Fixed vertical alignment issue by cropping flathub icon

![image](https://user-images.githubusercontent.com/19629102/47598731-08d32580-d9ba-11e8-8c3d-39ba30b93ef2.png)

And also I noticed that there are two separate heights for both mobile and desktop (56px and 64px) in original flathub store, So I also fixed this one.

splitt3r commented:
@nmanumr perhaps it would be nicer to use the https://github.com/angular/material2/blob/master/src/lib/toolbar/toolbar.scss#L4-L5 given variables for that.

jgarciao commented:
Hi Nauman! Thanks for contributing to Fly and fixing the little issues
Allan remarked. I'll try your code next week

El ds., 27 d’oct. 2018, 11:57, Kai Schröer <notifications@github.com> va
escriure:

> @nmanumr <https://github.com/nmanumr> perhaps it would be nicer to use
> the
> https://github.com/angular/material2/blob/master/src/lib/toolbar/toolbar.scss#L4-L5
> given variables for that.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/flathub/linux-store-frontend/pull/144#issuecomment-433607223>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEFTrhFeu2w-plbhwjk9pdzIPEP8xiZks5upC4agaJpZM4X3G14>
> .
>

jgarciao commented on src/app/shared/toolbar/toolbar.component.html:
```diff
@@ -1,75 +1,42 @@
-    <mat-menu #menu="matMenu">
+    <div class="hide-md toolbar-nav">
+      <button mat-button routerLink="apps">APPLICATIONS</button>
+      <button mat-button href="https://github.com/flathub/flathub/wiki/App-Submission">PUBLISH</button>
```
External links don't work with button. The code should be like this:

` <a mat-button href="https://github.com/flathub/flathub/wiki/App-Submission" aria-label="Publish an app on Flathub">PUBLISH</a>`

jgarciao commented on src/app/shared/toolbar/toolbar.component.html:
```diff
@@ -1,75 +1,42 @@
-      />
+      <mat-menu #menu="matMenu" overlapTrigger="false">
+        <button mat-menu-item routerLink="apps">APPLICATIONS</button>
+        <button mat-menu-item href="https://github.com/flathub/flathub/wiki/App-Submission">PUBLISH</button>
```
External links don't work with button. The code should be like this:

` <a mat-button href="https://github.com/flathub/flathub/wiki/App-Submission" aria-label="Publish an app on Flathub">PUBLISH</a>`

jgarciao commented:
I've deployed the new version in the [test sever.](http://45.55.104.129) It looks good to me. 

@allanday  If you don't say anything otherwise I will merge with master and deploy it into production next Thursday 15th.
